en/docs/using-wso2-identity-server/password-recovery.md

```diff
@@ -278,49 +270,47 @@ To try this out, first create a user in the Identity Server.
     
 
-    ![]( ../../assets/img/50514066/57747162.png) 
+    ![recover-with-security-questions](../../assets/img/using-wso2-identity-server/recover-with-security-questions.png) 
 
     !!! note
     
         In case you have enabled password recovery with reCaptcha
         verification, enter the required fields as specified above and
         select the recaptcha checkbox as well.
     
-        ![](https://lh3.googleusercontent.com/cDKsCGD5sa3R0MmEKOUd8wsysrEO_lCAdY1ETIzxgJ04xR1s7PzqmVgIyftst1Y_M002KwJDIk-LY7xXyIE2Bqi8b8nlB3z7o8S_TAMku54G97Xg2Es0a-qVeQnDDD4H78oGQt1A)   
+        ![password-recovery-recaptcha-verification](../../assets/img/using-wso2-identity-server/password-recovery-recaptcha-verification.png)   
     
 
       
 
 11. Enter the answers to the challenge questions and click **Submit**
     .  
-    ![]( ../../assets/img/50514066/57747166.png) 
+    ![submit-answers](../../assets/img/using-wso2-identity-server/submit-answers.png) 
+
 12. Enter a new password and click **Submit**. You will receive a
     notification of successful password reset.  
 
-![]( ../../assets/img/50514066/57747167.png) 
+![enter-new-password](../../assets/img/using-wso2-identity-server/enter-new-password.png) 
 
-![]( ../../assets/img/50514066/57747168.png) 
-
-  
+![successful-password-reset](../../assets/img/using-wso2-identity-server/successful-password-reset.png) 
 
 !!! tip
     
     For information on the REST APIs for password recovery using challenge
     questions, see [the swagger docs on Account Recovery REST
-    APIs](https://docs.wso2.com/display/IS530/apidocs/account-recovery/).
+    APIs](https://docs.wso2.com/display/IS580/apidocs/account-recovery/).
     
 
-**Related Links**
+!!! info "Related Links"
 
--   To set up reCaptcha for password recovery with secret questions, see
-    [Configuring Google reCaptcha for Security-Question Based Password
-    Recovery](_Configuring_Google_reCaptcha_for_Security-Question_Based_Password_Recovery_)
-    .
--   By default, the claim values of the identity claims used in this
-    feature are stored in the JDBC datasource configured in the
-    `            identity.xml           ` file. See [Configuring
-    Claims](https://docs.wso2.com/display/IS540/Configuring+Claims) for
-    more information on how to store the claim values in the user store.
+        -   To set up reCaptcha for password recovery with secret questions, see
+        [Configuring Google reCaptcha for Security-Question Based Password
+        Recovery](../../using-wso2-identity-server/configuring-google-recaptcha-for-security-question-based-password-recovery).
+        -   By default, the claim values of the identity claims used in this
+        feature are stored in the JDBC datasource configured in the
+        `            identity.xml           ` file. See [Configuring
+        Claims](../../using-wso2-identity-server/configuring-claims) for
+        more information on how to store the claim values in the user store.
```

Comment:
with the new config model, the claim values will not be in the identity.xml anymore right?